demo/src/pick_place_task.cpp

```diff
@@ -194,16 +201,17 @@ void PickPlaceTask::init() {
   ---- *               Generate Grasp Pose                *
 		 ***************************************************/
 		{
-			// Sample grasp pose
+			// Sample grasp pose candidates in angle increments around the z-axis of the object.
 			auto stage = std::make_unique<stages::GenerateGraspPose>("generate grasp pose");
 			stage->properties().configureInitFrom(Stage::PARENT);
 			stage->properties().set("marker_ns", "grasp_pose");
 			stage->setPreGraspPose(hand_open_pose_);
 			stage->setObject(object);
-			stage->setAngleDelta(M_PI / 12);
+			stage->setAngleDelta(M_PI / 4);
```

Comment:
Why did this value change?

If it was up to me, I would change this to `M_TAU/24` as per [this PR of mine](https://github.com/ros-planning/moveit_tutorials/pull/554), but let's not open that discussion.